### PR TITLE
Various test cleanups

### PR DESCRIPTION
**Pull request by h-m-m**
https://github.com/maebeale/mutual-aid/pull/75 is big, so I'm breaking things out of it

@exbinary's commit feels uncontroversial to me.

Also, we have an option to import `assert` in setup.js so you don't have to manually import it. I like removing the boilerplate. I'm worried it might confuse new people, and I'm sad that it seems to hamper some auto-complete functionality in Visual Studio Code — but only some. Any thoughts?

### CODE REVIEW

File: spec/factories/listings.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 FactoryBot.define do
   factory :listing do
     location
-    service_area
```
Comment (maebeale) [inline spec/factories/listings.rb:4]: per discussion, we still need this

File: spec/factories/listings.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 FactoryBot.define do
   factory :listing do
     location
```
Comment (maebeale) [inline spec/factories/listings.rb:3]: i'll check out how to make sure listing db/model don't require location

File: spec/factories/listings.rb (hunk 1 above)
Comment (h-m-m) [reply spec/factories/listings.rb:4]: @maebeale If it's OK with you, I want to merge this in first. Then we can add this back to the factory once the db supports it

File: spec/factories/listings.rb (hunk 2 above)
Comment (h-m-m) [reply spec/factories/listings.rb]: It's already optional in the db, so that's good. I'll update the factory accordingly

**Pull request merged**